server/routes/report.ts

```diff
@@ -214,9 +208,7 @@ export default function (router: IRouter) {
           }
         );
         return response.ok({
-          body: {
-            data: esResp,
-          },
+          body: esResp._source,
```

Comment:
question: does this change the response returned from the API at all? Or is it refactoring for more elegant API work in back-end?

Reply:
yes, it extracts only the useful data for client side, gets rid of the redundant data fields returned by ES, such as `seq_num`, `version_num`, `id`